src/styles/page/page-catalog.scss

```diff
@@ -0,0 +1,30 @@
+@import "./functions-catalog.scss";
+
+.card-holder {
```

Comment:
This is a part of card? Or catalog?

Reply:
its part of catalog, because I use this position only with this catalog.